format_pcre_tests.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/sh
```

Comment:
You can also do like
```
#!/usr/bin/jq -srf
```

Reply:
Nice!

Semi-relatedly, I'm thinking of scrapping this script and either adapting the `extract_pcre_tests.py` script to generate `retest` format directly or writing a small C program to do this (removing the python dependency).